tests/Type/GeneralTypeTest.php

```diff
@@ -74,6 +74,10 @@ public static function dataFileAsserts(): iterable
             yield from self::gatherAssertTypes(__DIR__ . '/data/query-builder-l10-24.php');
         }
 
+        if (version_compare(LARAVEL_VERSION, '10.27.0', '>=')) {
+            yield from self::gatherAssertTypes(__DIR__ . '/data/eloquent-builder-l10-27.php');
+        }
```

Comment:
We are adding the stubs to `10.0.0` anyway. So it'll be available for 10.x. No need to check for 10.27 specifically.

If there is a big change before and after 10.27, then you need to create a folder `10.27.0` in `stubs` directory and put the stubs there.